__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe("GET /api/topics", () => {
+  test("returns 200 status code and an array of objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(Array.isArray(response.body.topic)).toBe(true);
+        expect(response.body.topic.length).toBeGreaterThan(0);
```

Comment:
if you know the actual length of the topics array I would assert against it here, but making sure it's not empty is fine too